src/cbc_sdk/base.py

```diff
@@ -2124,28 +2138,36 @@ def _check_range(self, range):
             raise ApiError("`bucket_size` should be either int or ISO8601 timestamp string")
 
     def add_range(self, range):
-        """Sets the facet ranges to be received by this query.
+        """
+        Sets the facet ranges to be received by this query.
 
         Arguments:
             range (dict or [dict]): Range(s) to be received.
 
         Returns:
             Query (AsyncQuery): The Query object that will receive the specified range(s).
 
-        Note: The range parameter must be in this dictionary format:
+        Note:
+            The range parameter must be in this dictionary format:
+
             {
+
```

Comment:
Why the extra new lines here?

Reply:
Without the lines it prints as: 
“bucket_size”: “<object>”, “start”: “<object>”, “end”: “<object>”, “field”: “<string>”

instead of each on a new line